packages/web/src/components/gcds-link/gcds-link.tsx

```diff
@@ -0,0 +1,220 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Watch,
+  Prop,
+  State,
+  h,
+} from '@stencil/core';
+import { assignLanguage, observerConfig } from '../../utils/utils';
+import { inheritAttributes } from '../../utils/utils';
+import i18n from './i18n/i18n';
+
+@Component({
+  tag: 'gcds-link',
+  styleUrl: 'gcds-link.css',
+  shadow: true,
+})
+export class GcdsLink {
+  @Element() el: HTMLElement;
+
+  private shadowElement?: HTMLElement;
+
+  /**
+   * Props
+   */
+
+  /**
+   * Set the link size
+   */
+  @Prop({ mutable: true }) size: 'regular' | 'small' = 'regular';
```

Comment:
@daine @ethanWallace wondering if we should add `inherit` as a third option and set the `size` prop to default to `inherit`.

Reasoning for this:

The `regular` size currently gets its font size from the parent element, which works well for links inside a text block. However, when these links are used outside of a text block context (e.g. Resources links on the Storybook link overview page), they still inherit the font size from the parent. In this case, the links appear smaller than the expected regular font size.

To avoid this, we could add the `inherit` option as the default value for the `size` prop. The `inherit` option can then be used for links within headings or other text blocks. The `regular` option can be used for standalone links (e.g. the links in the resources section) and use a font token to load our regular text styling.

Reply:
Good idea! I am for it